### ISSUE TITLE
Just upgraded from 9.4.1 to 9.6.0 - file_line is no longer supported

### ISSUE THREAD

**Issue by djmills64** [puppetlabs/puppetlabs-stdlib#1438]
## Describe the Bug
After upgrading to 9.6.0 we are geting the error:
`ESC[1;31mError: Evaluation Error: Error while evaluating a Resource Statement, Unknown resource type: 'file_line' (file: /work/code/puppet/module/manifests/puppet_cron_client.pp, line: 597, column: 7) on node pentest-cloudexporter.ap-southeast-2.compute.internalESC[0m`

## Expected Behavior
We didn't get this error with 9.4.1

## Steps to Reproduce
1. Write some puppet code using the resource file_line
2. apply the code

## Environment
 - Version 7.33.0
 - Platform [e.g. Ubuntu 18.04]
'# uname -a
Linux pentest-cloudexporter 6.5.0-1024-aws #24~22.04.1-Ubuntu SMP Thu Jul 18 10:43:12 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux'

**Comment by ekohl** [comment]
That should still be there. Perhaps another case of [environment isolation](https://www.puppet.com/docs/puppet/8/environment_isolation.html) problems?

**Comment by djmills64** [comment]
Thanks, I'll investigate.

**Comment by djmills64** [comment]
Yes, that help. Interestingly, most of our sites didn't require this. So I'm not sure why one or 2 did.

**Comment by bwitt** [comment]
Sounds like this issue can be closed?
